test/suites/container_devices_disk.sh

```diff
@@ -87,3 +89,42 @@ test_container_devices_raw_mount_options() {
   # shellcheck disable=SC2154
   deconfigure_loop_device "${loop_file_1}" "${loop_device_1}"
 }
+
+test_container_devices_disk_ceph() {
+  local LXD_BACKEND
+
+  LXD_BACKEND=$(storage_backend "$LXD_DIR")
+  if ! [ "${LXD_BACKEND}" = "ceph" ]; then
+    return
+  fi
+
+  RBD_POOL_NAME=lxdtest-$(basename "${LXD_DIR}")-disk
+  ceph osd pool create "${RBD_POOL_NAME}" 1
+  rbd create --pool "${RBD_POOL_NAME}" --size 50MB my-volume
+  RBD_DEVICE=$(rbd map --pool "${RBD_POOL_NAME}" --name admin my-volume)
```

Comment:
Should be `rbd map --pool "${RBD_POOL_NAME}" my-volume`, we don't need a `--name admin` there.